Add 7 various missing recipes

GravesXI opened the pull request:
<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

I added a few recipes that wasn't added yet and tested to make sure they all worked
Flanged mace--- https://ffxiclopedia.fandom.com/wiki/Flanged_Mace
Zanbato--- https://ffxiclopedia.fandom.com/wiki/Zanbato?so=search
Flamberge--- https://ffxiclopedia.fandom.com/wiki/Flamberge?so=search
Silver Chain--- https://www.bg-wiki.com/ffxi/Silver_Chain
Silver Mail--- https://www.bg-wiki.com/ffxi/Silver_Mail
Death Scythe and desynth --- https://ffxiclopedia.fandom.com/wiki/Death_Scythe?so=search

NasouXI committed:
Add missing recipes

Xaver-DaRed commented:
Additional sources:
Flanged mace - http://wiki.ffo.jp/html/15470.html
Zanbato - http://wiki.ffo.jp/html/6881.html
Flamberge - http://wiki.ffo.jp/html/7787.html - Discrepancies between pages, but highest values used.
Silver Chain - http://wiki.ffo.jp/html/7244.html
Silver Mail - http://wiki.ffo.jp/html/10565.html
Death Scythe - http://wiki.ffo.jp/html/3914.html - Discrepancies between pages, but highest value used.

TeoTwawki reviewed (changes requested):
@bsm267 & @Xaver-DaRed 
I humbly request sources either in the commit (can `amend` the message) or in the file as comments so that future editors can tell how we decided what values to got with (and hopefully prevent them from going _"well the other wiki **I'm** looking at says..."_ and changing it in another PR).

NasouXI committed:
added info to the new recipes

Flanged mace--- https://ffxiclopedia.fandom.com/wiki/Flanged_Mace
Zanbato--- https://ffxiclopedia.fandom.com/wiki/Zanbato?so=search
Flamberge--- https://ffxiclopedia.fandom.com/wiki/Flamberge?so=search
Silver Chain--- https://www.bg-wiki.com/ffxi/Silver_Chain
Silver Mail--- https://www.bg-wiki.com/ffxi/Silver_Mail
Death Scythe and desynth --- https://ffxiclopedia.fandom.com/wiki/Death_Scythe?so=search

Additional sources:

Flanged mace - http://wiki.ffo.jp/html/15470.html
Zanbato - http://wiki.ffo.jp/html/6881.html
Flamberge - http://wiki.ffo.jp/html/7787.html - Discrepancies between pages, but highest values used.
Silver Chain - http://wiki.ffo.jp/html/7244.html
Silver Mail - http://wiki.ffo.jp/html/10565.html
Death Scythe - http://wiki.ffo.jp/html/3914.html - Discrepancies between pages, but highest value used
